src/proof/alethe/alethe_node_converter.cpp

```diff
@@ -17,23 +17,187 @@
 
 #include "expr/node_algorithm.h"
 #include "expr/skolem_manager.h"
+#include "proof/proof_rule_checker.h"
+#include "util/bitvector.h"
+#include "util/rational.h"
 
 namespace cvc5::internal {
 namespace proof {
 
-AletheNodeConverter::AletheNodeConverter(NodeManager* nm) : NodeConverter(nm) {}
+Node AletheNodeConverter::maybeConvert(Node n, bool isAssumption)
+{
+  d_error = "";
+  Node res = convert(n);
+  if (!d_error.empty())
+  {
+    return Node::null();
+  }
+  if (isAssumption)
+  {
+    d_convToOriginalAssumption[res] = n;
+  }
+  return res;
+}
 
 Node AletheNodeConverter::postConvert(Node n)
 {
   NodeManager* nm = NodeManager::currentNM();
   Kind k = n.getKind();
   switch (k)
   {
+    case Kind::BITVECTOR_BITOF:
+    {
+      std::stringstream ss;
+      ss << "(_ bitOf " << n.getOperator().getConst<BitVectorBitOf>().d_bitIndex
```

Comment:
Is this impacted by the renaming https://github.com/cvc5/cvc5/pull/10789?
Should this be updated already?

Reply:
That's a good point. This was added exactly to avoid using the *previous* renaming. The name to be use here is the name used in the Alethe spec. We are in the process of updating those names. I'll double check and update here accordingly.

Reply:
OK, I updated the names to how they are currently in the Alethe spec. Once this PR is merged, #10789 will have to be updated since the respective kinds are changing.